### PR TITLE
Goerli Migration

### PR DESCRIPTION
**Pull request by colbr**

### CODE REVIEW

File: src/lib/paymentToken.ts [hunk 1]
```diff
@@ -11,12 +12,13 @@ const getPaymentTokenInfo = async (
...
 	tokenId: string,
 ): Promise<ConvertedTokenInfo> => {
+	const { rinkeby, goerli } = ZContracts.zer0ProtocolAddresses;
 	if (paymentTokenCache[tokenId]) {
 		return paymentTokenCache[tokenId];
 	} else {
 		if (
-			tokenId.toLowerCase() ===
-			'0x3Ae5d499cfb8FB645708CC6DA599C90e64b33A79'.toLowerCase()
+			tokenId.toLowerCase() === rinkeby?.tokens?.wildToken?.toLowerCase() ||
```
Comment (domw30) [inline src/lib/paymentToken.ts:20]: Is there a reason to keep Rinkeby related configurations?

Comment (domw30) [inline src/lib/paymentToken.ts:20]: Is there a reason to keep Rinkeby related configurations?

**Review by domw30 — APPROVED** [review]
Changes look good to me Subbu

File: src/lib/paymentToken.ts (hunk 1 above)
Comment (padala) [reply src/lib/paymentToken.ts:20]: We can actually get rid of it because this never gets used. Will update it. Thanks

**Pull request merged**